src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java

```diff
@@ -1139,6 +1140,28 @@ public void shouldKeepEmptyUserWhenGdprConsentIsInvalid() {
                 .isEqualTo(User.builder().build());
     }
 
+    @Test
+    public void shouldReturnAddErrorToAuctionContextWhenPrivacyIsNotValid() {
+        // given
+        given(httpRequest.getParam("gdpr_consent")).willReturn("consent-value");
+
+        givenBidRequest(
+                builder -> builder
+                        .user(User.builder().build())
+                        .ext(mapper.valueToTree(ExtBidRequest.of(null))),
+                Imp.builder().build());
+
+        // when
+        factory.fromRequest(routingContext, 0L).result();
+
+        // then
+        @SuppressWarnings("unchecked")
```

Comment:
It's better to use this before @Test

Reply:
Joshua Bloch in Effective Java 3 said

"Always use the SuppressWarnings annotation on the smallest scope possible. Typically this will be a variable declaration or a very short method or constructor. Never use SuppressWarnings on an entire class. Doing so could mask critical warnings.

If you find yourself using the SuppressWarnings annotation on a method or constructor that’s more than one line long, you may be able to move it onto a local variable declaration. "

Reply:
don't think it is critical, up to you.